### DISCUSSION TITLE
Is the template tag SEO friendly?

### DISCUSSION THREAD

**Discussion by QJan84** [alpinejs/alpine#3280] [4. General]
Quick question: Is the template tag SEO friendly or is the content then not recognized? Has anyone here gathered experience?

**Comment by ekwoka** [comment]
Anything inside the template tag is treated as invisible. Bots will technically see it, but only the dumbest bots would look inside it for page content.

Some bots, like google's will execute javascript when scraping for SEO so if alpine will render things from the template during init without any user actions, then Google should see those things.
